responders/UmbrellaBlacklister/UmbrellaBlacklister.py

```diff
@@ -5,19 +5,32 @@
 import requests
 from datetime import datetime
 
+
 class UmbrellaBlacklister(Responder):
     def __init__(self):
         Responder.__init__(self)
-        self.integration_url = self.get_param('config.integration_url', None, "Integration URL Missing")
+        self.integration_url = self.get_param(
+            'config.integration_url', None, "Integration URL Missing")
 
     def run(self):
         Responder.run(self)
 
-        if self.get_param('data.dataType') == 'domain':
+        data_type = self.get_param('data.dataType')
+        ioc_types = {"domain": "domain", "url": "url","fqdn": "fqdn"}
```

Comment:
Why are you using a dict instead of list?

`ioc_types = ["domain", "url", "fqdn"]`